lib/cloudkeeper/aws/proto_helper.rb

```diff
@@ -0,0 +1,82 @@
+module Cloudkeeper
+  module Aws
+    # Module refining basic GRPC structs with additional methods
+    #   used for conversion from one format to another
+    module ProtoHelper
+      refine CloudkeeperGrpc::Appliance do
```

Comment:
Maybe you can do some functional programming magic and make this more dynamically generated.